source/module_hsolver/hsolver_lcao.cpp

```diff
@@ -54,6 +53,10 @@ void HSolverLCAO::solveTemplate(hamilt::Hamilt* pHamilt,
     }
     else if (this->method == "lapack")
     {
+        //We are not supporting diagonalization with lapack
+        //until the obsolete globalc::hm is removed from
+        //diago_lapack.cpp
+        /*
```

Comment:
This is related to compiling without MPI, can we ban this feature until Issue#1420 be solved?

Reply:
i think the lapack solver is already broken, unless someone has fixed it at some point ..